### PR TITLE
Make some shit

### PR DESCRIPTION
**Pull request by photonbit**

### CODE REVIEW

File: kw/skycrapper/settings.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 STALL_MAPPING = {
-    "FAIRY": 0,
-    "AIRWICK": 1,
-    "NESCAFE": 2,
-    "ARIEL": 3,
-    "FINISH": 4  # JUST FINISH ALREADY
+    "HARD_LUMPS": 0,
```
Comment (kantord) [inline kw/skycrapper/settings.py:2]: zero should be like "nothing" no? when nobody used the stall

Comment (photonbit) [reply kw/skycrapper/settings.py:2]: `None` should be nothing, `0` is the first thing :)

Comment (kantord) [reply kw/skycrapper/settings.py:2]: `None` is not a number. I bet any mathematician caring about shit (this time literally) would kill you for this suggestion.

Comment (photonbit) [reply kw/skycrapper/settings.py:2]: Lucky us, we are not making maths: https://www.cs.utexas.edu/users/EWD/transcriptions/EWD08xx/EWD831.html

**Pull request closed without merging**